ci-operator/step-registry/azure/provision/service-principal/minimal-permission/azure-provision-service-principal-minimal-permission-commands.sh

```diff
@@ -113,6 +118,9 @@ az --version
 
 # set the parameters we'll need as env vars
 AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
+if [[ -f "${CLUSTER_PROFILE_DIR}/installer-sp-minter.json" ]]; then
```

Comment:
Which cluster profile is using `installer-sp-minter.json` , `azure4` ?

Reply:
New SP is created under subscription "Openshift CI" and saved into `installer-sp-minter.json`, which has permission to create new SP, and this json file is saved in vault which is related with cluster profile `azure4`
Detailed discussion is [here](https://redhat-internal.slack.com/archives/C01V1DP387R/p1733830693275249).